About.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { StyleSheet, Text, View, Button } from 'react-native';
+import { Font } from 'expo';
+import Logo from './Logo';
+
+
+export class About extends React.Component {
+  state = {fontLoaded: false};
+  async componentDidMount() {
+    await Font.loadAsync({
+      'double-bubble-shadow': require('./assets/fonts/Double_Bubble_shadow.otf'),
+      'source-sans-pro': require('./assets/fonts/SourceSansPro-Bold.ttf'),
+    });
+    this.setState({fontLoaded: true});
+  }
+  render() {
+    return (
+      <View style={styles.container}>
+        <View style={{height: '70%', backgroundColor: '#C2515B'}}>
+            <Logo font={this.state.fontLoaded} />
+            <View style={styles.aboutBox}>
+            {
+              this.state.fontLoaded ? (
+                <Text style={styles.aboutText}>
+                PongMates is designed to connect Dartmouth students through the game of Pong.{"\n"}{"\n"}Created by three WISP Interns working with the DALI Lab.{"\n"}{"\n"}We hope you enjoy!
+                </Text>
+              ) : null
+            }
+            </View>
+        </View>
+        <View style={{
+          flex: 1,
+          justifyContent: 'flex-end',
+          alignItems: 'center',
+          marginBottom: 30,
+        }}>
+          <View style={[styles.button, {backgroundColor: '#FFC928'}]}>
```

Comment:
Space between curly brackets but not square brackets (so the square brackets are correct in this case!)